taskcluster/ci/build/kind.yml

```diff
@@ -0,0 +1,42 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - focus_android_taskgraph.transforms.build:transforms
+    - taskgraph.transforms.job:transforms
+    - taskgraph.transforms.task:transforms
+
+kind-dependencies:
+    - toolchain
+
+job-defaults:
+    fetches:
+        toolchain:
+            - android-sdk-linux
+            - android-gradle-dependencies
+    run:
+        using: gradlew
+        use-caches: false
+        workdir: '/builds/worker'
+    run-on-tasks-for: []
+    treeherder:
+        kind: build
+        symbol: B
+        platform: android-all/opt
+        tier: 1
+    worker-type: b-android
+    worker:
+        docker-image: {in-tree: base}
+        max-run-time: 7200
+        chain-of-trust: true
+
+jobs:
+    debug:
+        description: 'Build Focus/Klar for Android from source code'
+        run-on-tasks-for: [github-pull-request, github-push]
+        run:
+            gradle-build-type: debug
+            gradle-build-name: focusDebug
```

Comment:
If we can cut down the build time by running them in separate tasks (like we did in A-C) then I think I'd prefer that.

Same for all build types: If there are no other constraints (resource usage, cost, ..) then I would prefer to have all variants build for PRs (or at the very least when merged). It will make sure that we find beta/release-specific build errors immediately. They don't happen often, but if they happen right before a release then it's always super annoying.

Reply:
hrm, so if I run `assembleFocusDebug`, which is what those two lines of code above are doing 'under the hood', I see all variants are built, including klar. Is that expected? https://firefox-ci-tc.services.mozilla.com/tasks/MG6UAfEFQaefwhfGVIO58A/runs/0/logs/public/logs/live.log